zcutil/make-release.py

```diff
@@ -177,8 +177,27 @@ def patch_release_height(releaseheight):
 
 @phase('Building...')
 def build():
+    base_dir = os.getcwd()
+    depends_dir = os.path.join(base_dir, 'depends')
+    src_dir = os.path.join(base_dir, 'src')
     nproc = sh_out('nproc').strip()
-    sh_log('./zcutil/build.sh', '-j', nproc)
+    sh_progress([
+        'Staging boost...',
+        'Staging libevent...',
+        'Staging zeromq...',
+        'Staging libgmp...',
+        'Staging libsodium...',
```

Comment:
These are dependency build lines that may or may not occur, but will definitely occur in this order if at all.